test/core/channel/minimal_stack_is_minimal_test.cc

```diff
@@ -139,7 +137,7 @@ static int check_stack(const char* file, int line, const char* transport_name,
   }
   {
     grpc_core::ExecCtx exec_ctx;
-    GPR_ASSERT(grpc_core::CoreConfiguration::Get().channel_init().CreateStack(
+    EXPECT_TRUE(grpc_core::CoreConfiguration::Get().channel_init().CreateStack(
```

Comment:
This one should be `ASSERT_TRUE()`, but that will work only if you take my suggestion above such that `check_stacl()` returns void.

Reply:
Done.